### PR TITLE
Adjustments for VS 2022 Preview 4

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): https://github.com/unoplatform/uno/issues/7082

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

- Add `.vsconfig` changes to support the new .NET 6 workloads
- Downgrade the wasm template to net5 for https://github.com/unoplatform/uno/issues/7082 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/get-started-vsmac.md [hunk 1]
```diff
@@ -12,54 +12,52 @@ While it is easier to create apps using Uno Platform on Windows, you can also cr
...
-    - Right click to remove them from the solution.
-    - At the top of the tree, on the solution node, right click and select **Restore Nuget Packages**
-1. You can now run on iOS, Android, and macOS by setting your startup project and running.
-![startup-projects](Assets/quick-start/vs-mac-build.png)
+1. You can now run on iOS, Android, macOS and Skia.GTK projects by changing your startup project and start the debugger.
    
 Note: You will not be able to build the UWP and WPF projects on a Mac. All changes to this project must be made on Windows.
 
-## Create a new project using the command line
+## Create a other projects types using the command line
```
Comment (ajpinedam) [inline doc/articles/get-started-vsmac.md:35]:
```suggestion
## Create other projects types using the command line
```

File: src/SolutionTemplate/UnoSolutionTemplate.net6/Wasm/UnoQuickStart.Wasm.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net6.0</TargetFramework>
+		<TargetFramework>net5.0</TargetFramework>
```
Comment (ajpinedam) [inline src/SolutionTemplate/UnoSolutionTemplate.net6/Wasm/UnoQuickStart.Wasm.csproj:5]: was this change intended?

**Review by ajpinedam — APPROVED** [review]

File: doc/articles/get-started-vsmac.md [hunk 3]
```diff
@@ -12,54 +12,52 @@ While it is easier to create apps using Uno Platform on Windows, you can also cr
...
-3. In the terminal, navigate to your build output. This will typically be: `MyProject.Wasm > bin > Debug > netstandard2.0 > dist > server.py` Run the `server.py` program.
-4. In your browser, open localhost:8000. 
+1. Build the project
+1. In the terminal, navigate to your build output path. This will typically be: `MyProject.Wasm > bin > Debug > net5.0 > dist`
+1. Install `dotnet serve`:
+    ```
+    dotnet tool install -g dotnet-serve
+    ```
+1. Once installed type `dotnet serve`.
+4. Navigate to the url presented by the tool to run your application
```
Comment (Youssef1313) [inline doc/articles/get-started-vsmac.md:56]:
```suggestion
1. Navigate to the url presented by the tool to run your application
```

File: src/SolutionTemplate/UnoSolutionTemplate.net6/Wasm/UnoQuickStart.Wasm.csproj (hunk 2 above)
Comment (jeromelaban) [reply src/SolutionTemplate/UnoSolutionTemplate.net6/Wasm/UnoQuickStart.Wasm.csproj:5]: Yes, it's intended, there's an issue in VS2022 with net6 and the bootstrapper
